results/resultaten-versie-0.1.md

```diff
@@ -74,25 +74,50 @@ Nu de mails geïdentificeerd en geschoond zijn kunnen we de sentiment analyse er
+1.  [Dutch RoBERTa-based Language Model](https://arxiv.org/pdf/2001.06286.pdf) dit model maakt gebruikt van een ***boolean*** methode waarbij stukken tekst enkel als ***positief*** of ***negatief*** geclassificeerd worden (dus geen neutraal optie). Ook wordt er een betrouwbaarheidsscore toegewezen die loopt van ***0,5*** tot ***1***. Het gebruikte model is getraind op basis van een review dataset DBRD. Dit is een dataset die bestaat uit 118.516 boek reviews die vervolgens door gebruikers gescoord zijn. Voor het trainen van RoBERTa zijn 22.252 van deze reviews gebruikt. 
+2.  [Bert multilingual sentiment](https://arxiv.org/pdf/2007.13061.pdf) dit model maakt gebruik van een 5punts ***likert scale*** die loopt van heel negatief naar heel positief. Elk stuk tekst krijgt per punt op deze schaal een score tussen ***0.0*** en ***1.0***. Waarbij het totaal altijd 1 is, een stuk tekst kan bijv. 0,9 positief en 0,1 heel positief zijn. Deze data is getraind op Wikipedia-data dumps van 104 talen. 
+3.  [Robbert twitter sentiment](https://huggingface.co/btjiong/robbert-twitter-sentiment?text=a) Dit model is een afgeleide van het Roberta-based model. Daarbij is dat model als uitgangspunt genomen en vervolgens getraind op twitter data. Hier wordt gebruik gemaakt van een drie puntschaal netagtief, neutraal, positief.
+
    
->   Beide modellen kunnen [hier](https://huggingface.co/models?language=nl&sort=downloads&search=sentiment) gevonden worden.
+>   Alle modellen kunnen [hier](https://huggingface.co/models?language=nl&sort=downloads&search=sentiment) gevonden worden.
 
-## Resultaten 
-In de onderstaande figuur staan de resultaten van beide modellen. Er is een uitsplitsing gemaakt naar een aantal onderwerpen, daarbij is steeds naar de 15 meest voorkomende onderwerpen gekeken. Bij de analyse met het boolean model zijn alleen resulaten meegenomen met een betrouwbaarheid van > 0.95.
+## Betrouwbaarheidstests
+Alle modellen hebben we zelf getest door te kijken naar hotel reviews van een aantal hotels in Maastricht die we met een webscraper van [Tripadvisor.nl]('https://www.tripadvisor.nl/) gescraped hebben. We hebben in totaal naar 2.296 reviews van twee hotels gekeken. Deze reviews bevatten een korte review tekst en een review score van 1 tot 5. 
+We hebben vervolgens de review teksten door de drie verschillende modellen gehaald, daarnaast hebben we het 2e model (multilingual) omgezet naar een drie puntsschaal. Daarbij hebben we de steeds uiterste 2 waardes samen gevoegd (zeer positief wordt positief en idem voor negatief). Tot slot hebben we dit omgezette model samengoegd met het derde model (gemiddelde scores van de beide modellen samengevoegd). Hier kwamen de volgende resultaten uit voort:
+```ruby
+Accuracy test for sentiment analysis models on Tripadvisor data (n=2296) 
```

Comment:
Zou je deze info in een 3-by-3 tabel kunnen weergeven? Dat leest waarschijnlijk super chill af!